src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_with_native_pubsub.cs

```diff
@@ -0,0 +1,158 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.AcceptanceTests.ScenarioDescriptors;
+    using NUnit.Framework;
+
+    public class When_publishing_to_scaled_out_subscribers_with_native_pubsub : NServiceBusAcceptanceTest
```

Comment:
how about: `When_publishing_to_scaled_out_subscribers_on_multicast_transports`?


Or should we have a `/Rounting/Multicast` folder instead?